### PR TITLE
Hooks support for Deploy CLI Tool

### PR DESCRIPTION
**Pull request by rkhoriander**
- https://docs-content-staging-pr-8736.herokuapp.com/docs/extensions/deploy-cli
- https://docs-content-staging-pr-8736.herokuapp.com/docs/extensions/deploy-cli/guides/install-deploy-cli
- https://docs-content-staging-pr-8736.herokuapp.com/docs/extensions/deploy-cli/guides/import-export-directory-structure
- https://docs-content-staging-pr-8736.herokuapp.com/docs/extensions/deploy-cli/guides/import-export-yaml-file
- https://docs-content-staging-pr-8736.herokuapp.com/docs/extensions/deploy-cli/references/whats-new

### CODE REVIEW

**Review by shawnmclean — DISMISSED** [review]

**Comment by charlesjohnson** [comment]
On https://docs-content-staging-pr-8736.herokuapp.com/docs/extensions/deploy-cli, in the "Upgrading to Deploy CLI Tool v4" box:

Instead of hand-modifying the permissions on the deploy-cli application, can the customer pick up the new permissions by upgrading their deploy-cli extension to the latest version? This would be a simpler set of instructions, and customer could then verify that it worked correctly by looking for the additional permissions on the deploy-cli application.

**Comment by charlesjohnson** [comment]
On https://docs-content-staging-pr-8736.herokuapp.com/docs/extensions/deploy-cli/guides/install-deploy-cli, I believe installing the deploy-cli extension now takes care of all of the application configuration steps. Can someone else check me on this?

**Comment by charlesjohnson** [comment]
Everything else looks good.

File: articles/extensions/deploy-cli/guides/create-deploy-cli-application-manually.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
+---
```
Comment (erjohnson) [inline articles/extensions/deploy-cli/guides/create-deploy-cli-application-manually.md:1]: Add this to the sidebar nav?

**Review by erjohnson — APPROVED** [review]

**Comment by charlesjohnson** [comment]
Looks great thanks

**Pull request merged**